tests/com.espressif.idf.ui.test/configs/default-test.properties

```diff
@@ -1,16 +1,25 @@
 # Default build wait before the timeout exception is thrown by the test
-default.project.build.wait=800000
+default.project.build.wait=90800000
 
 # Default project copy delay before the timeout exception is thrown
 default.project.copy.wait=7000
 
 # Default paths for esp env configs (These may need to be updated as per Automation requirements and the env being used)
-default.env.esp.idf.path=C:\\Users\\aliaz\\Documents\\DevTools\\ESP-IDF\\esp-idf-v4.2.1
-default.env.esp.git.path=C:\\Program Files\\Git\\bin\\git.exe
+
+#Github Actions Path
+default.env.esp.idf.path=/opt/actions-runner/_work/idf-eclipse-plugin/idf-eclipse-plugin/dependencies/idf-tools
+default.env.esp.git.path=git
 default.env.esp.python.version=3.9
-default.env.esp.python.path=C:\\Program Files\\Python39\\python.exe
+default.env.esp.python.path=python3.9
+default.env.esp.idf.download.path=/opt/actions-runner/_work/idf-eclipse-plugin/idf-eclipse-plugin/
+
 
-default.env.esp.idf.download.path=C:\\esp-idf-download
+#Developer Path
+#default.env.esp.idf.path=C:\\Users\\aliaz\\Documents\\DevTools\\ESP-IDF\\esp-idf-v4.2.1
+#default.env.esp.git.path=C:\\Program Files\\Git\\bin\\git.exe
+#default.env.esp.python.version=3.9
+#default.env.esp.python.path=C:\\Program Files\\Python39\\python.exe
+#default.env.esp.idf.download.path=C:\\Users\\aliaz\\esp-idf-download
 
-default.download.url.stable=https://dl.espressif.com/dl/idf-eclipse-plugin/updates/latest/
-default.download.url.beta=https://dl.espressif.com/dl/idf-eclipse-plugin/updates/beta/
+#default.download.url.stable=https://dl.espressif.com/dl/idf-eclipse-plugin/updates/latest/
+#default.download.url.beta=https://dl.espressif.com/dl/idf-eclipse-plugin/updates/beta/
```

Comment:
Could we remove if this code is not required?

Reply:
Done